expression/builtin_time_vec_test.go

```diff
@@ -78,6 +79,13 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 			constants: []*Constant{{Value: types.NewStringDatum("2019-11-11"), RetType: types.NewFieldType(mysql.TypeString)}},
 		},
 	},
+	ast.TimeLiteral: {
+		{retEvalType: types.ETDuration, childrenTypes: []types.EvalType{types.ETDuration},
+			constants: []*Constant{
+				{Value: types.NewDurationDatum(types.Duration{Duration: time.Hour + time.Second}),
+					RetType: types.NewFieldType(mysql.TypeDuration)}},
```

Comment:
Why the type is `Duration` , but not `String`?  I think `TimeLiteral` function sig is used to convert String to Duration. So the input should be a string?

Reply:
```
func (b *builtinTimeLiteralSig) evalDuration(row chunk.Row) (types.Duration, bool, error) {
	return b.duration, false, nil
}
```

This is the original builtin function. I think it should be a Duration?

Reply:
@wshwsh12  am I correct?

Reply:
I don't think so. We can see timeLiteralFunctionClass.getFunction().
```
con, ok := args[0].(*Constant)
	if !ok {
		panic("Unexpected parameter for time literal")
	}
	dt, err := con.Eval(chunk.Row{})
	if err != nil {
		return nil, err
	}
	str := dt.GetString()
	if !isDuration(str) {
		return nil, types.ErrWrongValue.GenWithStackByArgs(types.DateTimeStr, str)
	}
	duration, err := types.ParseDuration(ctx.GetSessionVars().StmtCtx, str, types.GetFsp(str))

```
It use GetString and ParseDuration to get duration.

Reply:
Ohhh. I see. Should like this:
```
	ast.TimeLiteral: {
		{retEvalType: types.ETDuration, childrenTypes: []types.EvalType{types.ETString},
			constants: []*Constant{
				{Value: types.NewStringDatum("838:59:59"), RetType: types.NewFieldType(mysql.TypeString)}},
		},
	},
```

right?